paper.qmd

```diff
@@ -821,6 +875,11 @@ Sustainable transport policies should plan for walking, wheeling and cycling, an
 This raises the question of whether other active modes should be incorporated into the results using the OD-based approach outlined in this paper, or whether different modeling approaches are needed to properly capture the shorter distance trips typically made by walking [@cooper2018].
 Furthermore, the estimates of cycling presented in the CRUSE Tool omit multi-modal trips including public transport, and omit trip chaining, due to the need to capture a large portion of cycling potential within the resource constraints of the project.
 
+A final limitation of the approach relates to *how* it was developed.
```

Comment:
I like it, you recognize the participatory process could have been better and explain how. Perhaps it would be good to include some reference as to why the broader the better the participation. Although, I don't think it's essential either, since it's quite obvious.